Dockerfile-tensorflow-ngc

```diff
@@ -0,0 +1,32 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+# NGC images contain user owned files in /usr/lib
+RUN chown root:root /usr/lib
+
+# Copy various shell scripts that group dependencies for install
+COPY dockerfile_scripts /tmp/det_dockerfile_scripts
+
+RUN /tmp/det_dockerfile_scripts/install_deb_packages.sh
+
+# We uninstall these packages after installing. This ensures that we can
+# successfully install these packages into containers running as non-root.
+# `pip` does not uninstall dependencies, so we still have all the dependencies
+# installed.
+RUN python -m pip install determined && python -m pip uninstall -y determined
+
+RUN python -m pip install -r /tmp/det_dockerfile_scripts/notebook-requirements.txt \
+    -r /tmp/det_dockerfile_scripts/additional-requirements.txt
+
+ENV JUPYTER_CONFIG_DIR=/run/determined/jupyter/config
+ENV JUPYTER_DATA_DIR=/run/determined/jupyter/data
+ENV JUPYTER_RUNTIME_DIR=/run/determined/jupyter/runtime
+
+RUN /tmp/det_dockerfile_scripts/add_det_nobody_user.sh
```

Comment:
you addressed my review comments in one file but not the other; you added libnss_determined to the pytorch-ngc but not this tensorflow-ngc.  Please fix

Also be much more careful about these details.  It is not the reviewer's job to keep your code in sync for you.